File: conditionals/time_of_day.py
# Douglas London, time of day python

import time
current = time.time()
local_time = time.localtime(current)
hour = local_time.tm_hour # millitary time (0-23)

if(hour < 12):
    print("Good morning!")
elif(hour < 17):
    print("Good afternoon!")
else:
    print("Good evening!")

File: conditionals/time_of_day.c
// Douglas London, Time of day C
#include <stdio.h>
#include <time.h>





int main(void){
    

    time_t now = time(NULL);
    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;

    if(hour < 12){
    printf("Good morning!");
    }else if(hour < 17){
    printf("Good afternoon!");
}else{
    printf("Good evening!");
}






    return 0;
}